docs/reference/server-configuration.md

```diff
@@ -714,7 +824,7 @@ request can be useful when throwing an error is preferable to doing the potentia
 
 ## `ksql.query.pull.interpreter.enabled`
 
-**Per query:** yes
+**Per query:** yes (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

Comment:
@rodesai also calls out being configurable per-query explicitly in the explanation, so I'm going with "yes"
```suggestion
**Per query:** yes
```